eventiq_asb/broker.py

```diff
@@ -123,57 +121,28 @@ async def sender(
         consumer: Consumer,
         send_stream: MemoryObjectSendStream[ServiceBusReceivedMessage],
     ) -> None:
-        receiver = self.client.get_subscription_receiver(
-            topic_name=self.topic_name, subscription_name=consumer.topic
-        )
-        max_wait_time = consumer.options.get("max_wait_time", 5)
-        async with receiver, send_stream:
+        async with send_stream:
             while True:
-                batch = consumer.concurrency - len(
-                    send_stream._state.buffer  # noqa: SLF001
-                )
-                if batch == 0:
+                if self.msgs_queues[consumer.topic].empty():
                     await anyio.sleep(0.1)
                     continue
-
-                received_msgs = await receiver.receive_messages(
-                    max_message_count=batch, max_wait_time=max_wait_time
-                )
-                self.logger.debug("Fetching %d messages", batch)
-                for msg in received_msgs:
-                    self._receivers[id(msg)] = (
-                        receiver  # store weak reference to receiver for ack/nack
-                    )
-                    await send_stream.send(msg)
+                raw_message = await self.msgs_queues[consumer.topic].get()
```

Comment:
if we already overwrite service, maybe we can get msgs from msgs_queues directly in receiver instead of moving them from one queue to other?